### PR TITLE
WIP The Yearly Cleanup

### PR DESCRIPTION
**Pull request by Malwine**
- [x] Move sponsor logos to sponsor page
- [x] Add group picture of last workshop
- [x] Update information on top of the page

### CODE REVIEW

**Comment by socksy** [comment]
...did we forget to take a group photo, or did I just miss that?

**Comment by Malwine** [comment]
@socksy We took it. Julia has it.

File: index.html [hunk 1]
```diff
@@ -289,7 +289,7 @@
     <div class="row">
       <div class="col-md-6">
         <p><strong>Workshop January 2016</strong></p>
-        <p>The January workshop took place on 22nd and 23 January in Berlin. We had 32 students and around 20 coaches.</p>
+        <p>The January workshop took place on 22nd and 23rd January in Berlin. We had 32 students and around 20 coaches.</p>
```
Comment (Malwine) [inline index.html:292]: 👍 🇬🇧 📅

File: index.html [hunk 2]
```diff
@@ -12,17 +12,15 @@
           <b>English: </b> <a href="http://www.clojurebridge.org/">ClojureBridge</a>
           is a free Clojure programming workshop for women* -
           especially for those with little to no programming experience.<br>
-          <b>Next workshop: June 17 and 18, 2016.</b><br>
-          Applications are closed.
+          <b>Next workshop: presumably January 2017</b><br>
```
Comment (vsmart) [inline index.html:15]: I think the date I've heard floating around is November/Dec (before xmas) :) But I suppose we'll come up with a date (or a rough range) at retro maybe, and we can change it then.

**Comment by vsmart** [comment]
Thanks for doing this ❤️ 👏 @Malwine @socksy 👏 ❤️

**Pull request merged**